definitions/infra-awselb/golden_metrics.yml

```diff
@@ -28,6 +30,7 @@ latency:
       eventName: entityName
 errorRate:
   title: Error Rate
+  unit: OPERATIONS_PER_SECOND
```

Comment:
Should this not be a percentage?

Reply:
yes, that makes sense. good catch - i'll update.

Reply:
@cianBuckley  i um... deleted the original file/s off my machine (newb mistake, sorry!) 
is it possible to push this and i'll re-start the process and push this immediately following?